frontend/src/pages/RunList.test.tsx

```diff
@@ -398,6 +397,24 @@ describe('RunList', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('adds metrics column in the same order metrics were defined', async () => {
+    mockNRuns(1, {
+      run: {
+        metrics: [
+          { name: 'z_metric', number_value: 2 },
+          { name: 'y_metric', number_value: 1 },
+        ],
+      },
+    });
+    const props = generateProps();
+    tree = TestUtils.mountWithRouter(<RunList {...props} />);
+    await TestUtils.flushPromises();
+    tree.update();
+
+    const metricValues = tree.find("[data-testid='metric']");
+    expect(metricValues).toMatchSnapshot();
```

Comment:
Nit: always use toMatchInlineSnapshot

suggest also reduce some noise in the snapshot by

```
expect(metricValues.map(v => v.textContent)).toMatchInlineSnapshot()
```

Reply:
Refactored to compare with an Array instead. PTAL